lib/jack_tokenizer.rb

```diff
@@ -63,4 +86,16 @@ def int_val
   def string_val
     @current_token[1...-1]
   end
+
+  private
+
+  LETTERS = ("a".."z").to_a + ("A".."Z").to_a
```

Comment:
The `LETTERS` constant could live alongside `DIGITS`, `WHITESPACES` etc if you liked.